src/table/utils/color-utils.ts

```diff
@@ -46,8 +46,8 @@ export function resolveToRGBAorRGB(input: string): string {
   return 'none';
 }
 
-export function isDarkColor(color: string | undefined): boolean {
-  if (color === undefined) return false;
+export function isDarkColor(color: string | undefined | null): boolean {
+  if (color == null) return false;
```

Comment:
`undefined == null` is true, not sure if there is any side effect.

Reply:
that is the intention, so it catches both undefined and null

Reply:
Yes, i know. (color === undefined && color === null) is more common. But it should be fine